garnbarn_api/tests/test_tag_serializer.py

```diff
@@ -0,0 +1,50 @@
+from django.test import TestCase
+from garnbarn_api.models import Tag
+from garnbarn_api.serializer import CreateTagApiSerializer
+
+
+class TagAPITest(TestCase):
```

Comment:
Update the Class name, This is the test for Serializer

Reply:
Change done in commit d34d7a6.